hangman.rb

```diff
@@ -0,0 +1,192 @@
+# Brandi & Michelle - HANGMAN (Lunch + 2/3 Dinner)
+require "colorize"
+
+class Hangman
+
+  def initialize
+
+    @answer_bank = ["cupcake", "spaghetti", "cookie", "hamburger", "pizza", "sandwich", "crackers", "salad", "cheese", "burrito"] # possible hangman words
+    @guesses = 4 # number of allowed incorrect guesses
+    @num_correct = 0 # counter to compare to length of answer
+    @current_answer = @answer_bank[rand(@answer_bank.length)] # string (of answer)
+    @current_answer_array = @current_answer.split(//) # array (of answer)
+    @fillable_answer = [] # array (of blanks to be filled in by answer)
+    @alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"] # acceptable input
+    @already_guessed = [] # array (to store already guessed letters)
+
+    @current_answer_array.length.times do |x|
+      @fillable_answer[x] = " _ "
+    end
+
+    draw_art
+    print_word
+    play
+  end # initialize
+
+  # DRAWS THE ASCII ART
+  def draw_art
+    # parts of our hangman to be drawn
+    head = "(_)".colorize(:red)
+    arms = "\\|/".colorize(:light_black)
+    torso = "|".colorize(:purple)
+    legs = "/ \\".colorize(:magenta)
+
+    art = <<-END
+|     _________
+|     |/      |
+|     |
+|     |
+|     |
+|     |
+|     |
+|_____|___________
+
+    END
```

Comment:
Yeah, I was thinking there was too many of pictures of gallows, and wanted to DRY out the code, but didn't have time to figure it out.
So you can make multiple variables equal the same thing by: head = arms = torso = legs = "" ?